### PR TITLE
WW-520 fixed margins

### PR DESCRIPTION
**Pull request by cormorando**

### CODE REVIEW

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -325,6 +326,90 @@
...
 	}
+
+	.cnw-select {
+		margin: $content-spacing-large 0;
+
+		&.cnw-select-language {
+			margin-top: 0;
+
+			h3 {
+				margin-top: 0;
```
Comment (cormorando) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss:337]: This line is important cause there's no `!important` any more

File: extensions/wikia/CreateNewWiki/templates/CreateNewWiki_Index.php [hunk 2]
```diff
@@ -83,25 +83,21 @@
 			<h2><?= wfMessage( 'cnw-desc-headline' ) ?></h2>
 			<p class="creative"><?= wfMessage( 'cnw-desc-creative' )->escaped() ?></p>
 			<form name="desc-form" class="clearfix">
-				<div class="desc-block">
-					<textarea id="Description" placeholder="<?= wfMessage( 'cnw-desc-placeholder' )->escaped() ?>"></textarea>
-					<div class="checkbox" id="all-ages-div"
-						<?= (empty( $selectedLang ) || $selectedLang === $params['LangAllAgesOpt']) ? '' : 'style="display: none"' ?>>
-						<label>
```
Comment (lukaszkonieczny) [inline extensions/wikia/CreateNewWiki/templates/CreateNewWiki_Index.php:90]: Removing this label made text near checkbbox not clickable

**Review by lukaszkonieczny — CHANGES_REQUESTED** [review]

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 3]
```diff
@@ -325,6 +326,90 @@
...
+			}
+		}
+
+		&.cnw-select-vertical {
+			margin-bottom: 0;
+		}
+
+		.wds-dropdown {
```
Comment (idradm) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Could you rewrite this whole block to make it less nested?

**Review by idradm — CHANGES_REQUESTED** [review]
I've run linter on the file, could we fix whatever is possible at the current stage?

```
CreateNewWiki.scss:10:9 [W] ImportPath: Imported partial `extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index.scss` should be written as `extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index`
CreateNewWiki.scss:11:9 [W] ImportPath: Imported partial `extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons.scss` should be written as `extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/buttons`
CreateNewWiki.scss:40:25 [W] ColorVariable: Color literals like `rgba(26, 26, 26, 0.2)` should only be used in variable declarations; they should be referred to via variable everywhere else.
CreateNewWiki.scss:50:1 [W] IdSelector: Avoid using id selectors
CreateNewWiki.scss:52:2 [W] PropertySortOrder: Properties should be ordered background, border, box-sizing, color, font-family, font-size, line-height, margin, padding, position, width
CreateNewWiki.scss:72:3 [W] BorderZero: `border: 0` is preferred over `border: none`
CreateNewWiki.scss:75:11 [W] Shorthand: Shorthand form for property `margin` should be written more concisely as `0 0 $content-spacing-regular` instead of `0 0 $content-spacing-regular 0`
CreateNewWiki.scss:95:1 [W] QualifyingElement: Avoid qualifying attribute selectors with an element.
CreateNewWiki.scss:96:4 [W] PlaceholderInExtend: Prefer using placeholder selectors (e.g. %some-placeholder) with @extend
CreateNewWiki.scss:97:4 [W] PlaceholderInExtend: Prefer using placeholder selectors (e.g. %some-placeholder) with @extend
CreateNewWiki.scss:114:1 [W] QualifyingElement: Avoid qualifying attribute selectors with an element.
CreateNewWiki.scss:116:3 [W] BorderZero: `border: 0` is preferred over `border: none`
CreateNewWiki.scss:124:1 [W] QualifyingElement: Avoid qualifying attribute selectors with an element.
CreateNewWiki.scss:169:1 [W] QualifyingElement: Avoid qualifying attribute selectors with an element.
CreateNewWiki.scss:310:38 [W] LeadingZero: `0.1` should be written without a leading zero as `.1`
CreateNewWiki.scss:329:1 [W] IdSelector: Avoid using id selectors
CreateNewWiki.scss:329:1 [W] IdSelector: Avoid using id selectors
CreateNewWiki.scss:334:1 [W] IdSelector: Avoid using id selectors
CreateNewWiki.scss:354:1 [W] IdSelector: Avoid using id selectors
CreateNewWiki.scss:437:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
CreateNewWiki.scss:444:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
CreateNewWiki.scss:449:1 [W] QualifyingElement: Avoid qualifying class selectors with an element.
CreateNewWiki.scss:449:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
CreateNewWiki.scss:449:1 [W] MergeableSelector: Merge rule `li.spacer` with rule on line 444
CreateNewWiki.scss:454:1 [W] SelectorDepth: Selector should have depth of applicability no greater than 4, but was 5
CreateNewWiki.scss:455:6 [W] PropertySortOrder: Properties should be ordered background, color
```

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 4]
```diff
@@ -7,8 +7,8 @@
 @import 'skins/shared/mixins/columns';
 @import 'skins/shared/mixins/transform';
 @import 'extensions/wikia/ThemeDesigner/css/ThemeTab';
-@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index.scss';
-@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons.scss';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-variables/index';
+@import 'extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_buttons';
```
Comment (idradm) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: You can remove "_" (underscore) as well.

**Review by idradm — APPROVED** [review]

**Review by lukaszkonieczny — APPROVED** [review]

**Pull request merged**